velox/type/Filter.h

```diff
@@ -56,18 +57,23 @@ enum class FilterKind {
   kHugeintRange,
 };
 
+class Filter;
+using FilterUniquePtr = std::unique_ptr<Filter>;
```

Comment:
@mbasmanova points out the convention is to use `FilterPtr`.  Can we use this name?